### PR TITLE
#42 Two factor authentication

### PR DESCRIPTION
**Pull request by Muhet**
#### What does this PR do?
Seller login with higher security

#### Description of Task to be completed?

Users that require a higher level of security (Sellers) should be able to turn on 2FA where authentication to the application also incudes a an extra factor i.e SMS or Authentications codes.
After a successful login, Sellers should be redirected to their dashboard

#### How should this be manually tested?
```
git clone https://github.com/atlp-rwanda/ecommerce-app-predators-fe.git
cd ecommerce-app-predators-fe
git checkout ft-2FA-42
npm i
npm run dev
```

#### What are the relevant pivotal tracker/Trello stories?
Trello [Link]() (#-42)

#### Screenshots (if appropriate)
twofactor

### CODE REVIEW

**Review by faid-terence — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -54,5 +54,7 @@ Contributions are welcome! If you have any suggestions or find any bugs, please
 Vite
 React
 Tailwind CSS
+<<<<<<< HEAD
+=======
```
Comment (yobuir) [inline README.md]: You can remove this ` <<<<<<< HEAD `

**Review by yobuir — CHANGES_REQUESTED** [review]

File: src/redux/action/otpAction.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+const API_URL = 'http://localhost:3000/auth/';
```
Comment (yobuir) [inline src/redux/action/otpAction.tsx]: are we using local url or hosted one?

**Review by vanessa-abewe — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Muhet) [reply README.md]: I removed it

**Review by yobuir — APPROVED** [review]

File: src/redux/action/otpAction.tsx (hunk 2 above)
Comment (ALVINdimpos) [inline src/redux/action/otpAction.tsx:3]: can can u use hosted link

**Review by ALVINdimpos — APPROVED** [review]

File: src/redux/action/otpAction.tsx (hunk 2 above)
Comment (ALVINdimpos) [inline src/redux/action/otpAction.tsx]: try by using using hosted link

**Review by ALVINdimpos — CHANGES_REQUESTED** [review]

File: src/redux/action/otpAction.tsx (hunk 2 above)
Comment (Muhet) [reply src/redux/action/otpAction.tsx]: No I replaced it with the hosted one. check it

Comment (Muhet) [reply src/redux/action/otpAction.tsx]: I did check it again

**Review by ALVINdimpos — APPROVED** [review]

**Review by chanceDira — APPROVED** [review]
Good job

**Pull request merged**